packages/@react-aria/dnd/docs/useDrag.mdx

```diff
@@ -0,0 +1,375 @@
+import packageData from '@react-aria/dnd/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Drag and Drop
+keywords: [drag, drop, dnd, drag and drop, aria, accessibility]
+---
+
+# useDrag
+
+<PageDescription>{docs.exports.useDrag.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useDrag']} />
+
+## API
+
+<FunctionAPI function={docs.exports.useDrag} links={docs.links} />
+
+## Introduction
+
+Drag and drop is a common UI interaction that allows users to transfer data between two locations by directly moving a visual representation on screen. It is a flexible, efficient, and intuitive way for users to perform a variety of tasks, and is widely supported across both desktop and mobile operating systems.
+
+React Aria supports traditional mouse and touch based drag and drop, but also implements keyboard and screen reader friendly interactions. Users can press <Keyboard>Enter</Keyboard> on a draggable element to enter drag and drop mode. Then, they can press <Keyboard>Tab</Keyboard> to navigate between drop targets, and <Keyboard>Enter</Keyboard> to drop or <Keyboard>Escape</Keyboard> to cancel. Touch screen reader users can also drag by long pressing to activate drag and drop mode, swiping between drop targets, and double tapping to drop.
+
+See the [drag and drop introduction](dnd.html) to learn more.
+
+## Example
+
+This example shows how to make a simple draggable element that provides data as plain text. In order to support keyboard and screen reader drag interactions, the element must be focusable and have an ARIA role (in this case, `button`). While it is being dragged, it is displayed with a dimmed appearance by applying an additional CSS class.
+
+```tsx example
+import {useDrag} from '@react-aria/dnd';
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'
+      }];
+    }
+  });
+
+  return (
+    <div {...dragProps} role="button" tabIndex={0} className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+
+<Draggable />
+<DropTarget />
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.draggable {
+  display: inline-block;
+  vertical-align: top;
+  border: 1px solid gray;
+  padding: 10px;
+}
+
+.draggable.dragging {
+  opacity: 0.5;
+}
+
+.droppable {
+  width: 100px;
+  height: 80px;
+  border-radius: 6px;
+  display: inline-block;
+  padding: 20px;
+  margin-left: 20px;
+  border: 2px dotted gray;
+  white-space: pre-wrap;
+}
+
+.droppable.target {
+  border: 2px solid var(--blue);
+}
+```
+
+</details>
+
+### DropTarget
+
+The `DropTarget` component used above is defined below. See [useDrop](useDrop.html) for more details and documentation.
```

Comment:
```suggestion
The code for the `DropTarget` component used above is defined below. See [useDrop](useDrop.html) for more details and documentation.
```

Reply:
Other instances of it if you decide to take this change

Reply:
Not sure this is necessary. I think it's pretty clear that "is defined below" means code?